test/gpu/native/rocmOnly/compilerDriver/compilerDriver.good

```diff
@@ -0,0 +1,2 @@
+warning: --compiler-driver is overridden with --no-compiler-driver for AMD GPUs to work around a bug for the time being
```

Comment:
If you change the error message, remember to change this, too. (Been there done that)